src/services/users.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+let users = [];
+
+const init = () => {
+  users = [];
+};
+
+const getUsers = () => {
+  return users;
+};
+
+const getUser = (userId) => {
+  return users.find(user => user.id === +userId);
```

Comment:
`const foundUser = users.find(user => user.id === +userId);

return foundUser || null;
}`

Reply:
its could be better